### PR TITLE
Add test for recommender model

### LINKED ISSUE #145: Add test for recommender model
Tests should be created once the recommender model is factored out.

Depends on #49

### PR DESCRIPTION
**Pull request by NoB0**
Fixes #145

### CODE REVIEW

**Review by IKostric — APPROVED** [review]
LGTM

File: tests/mocks/mock_ontology.py [hunk 1]
```diff
@@ -3,13 +3,25 @@
...
 }
 
 
 class MockOntology:
     def __init__(self, path: str = None):
         self.ontology = self.load_ontology()
         self.slots_annotation = self.ontology["slots_annotation"]
+        self.agent_requestable = self.ontology["system_requestable"]
+        self.user_requestable = self.ontology["user_requestable"]
+        self.multiple_values_CIN = self.ontology["multiple_values"]
```
Comment (WerLaj) [inline tests/mocks/mock_ontology.py:24]: What does _CIN stand for?

**Review by WerLaj — APPROVED** [review]
LGTM

File: tests/mocks/mock_ontology.py (hunk 1 above)
Comment (NoB0) [reply tests/mocks/mock_ontology.py:24]: Current information need

**Pull request merged**